PatternPal/PatternPal.SyntaxTree/Abstractions/Members/IConstructor.cs

```diff
@@ -4,14 +4,27 @@
 
 namespace PatternPal.SyntaxTree.Abstractions.Members
 {
+    /// <summary>
+    /// An <see cref="INode"/> which represents a constructor.
+    /// </summary>
     public interface IConstructor : IMember, IParameterized, IBodied, IChild<IClass>
     {
         string GetConstructorType();
 
+        /// <summary>
+        /// Returns the arguments passed to constructor when it was invoked.
+        /// TODO check whether this is correct, or just delete as it isn't used.
```

Comment:
What is this todo about?

Reply:
And aren't these just the parameters of the constructor? Just like a method can have parameters, so can a constructor.